src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int backedCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator`. It's a good practice to make fields `private` and access them through getters and setters, which you have already done. Just ensure that this encapsulation principle is maintained if you add more fields in the future.